src/identity/email-password-identity-provider/tempConfiguration.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable id-length */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable max-len */
```

Comment:
Does the linter allow file-wide overrides? I don't think we want those.

Reply:
It does allow file overrides. But, these only exist to make it easy for me to get things running quickly. They will be removed.

Having said that, many elements of `node-oidc-provider` use snake case, so if there isn't a file override, there will be a lot of `// eslint-disable-next-line @typescript-eslint/naming-convention`

Reply:
Yeah we might want a folder-wide override indeed.